server/services/meilisearch/config.js

```diff
@@ -94,12 +94,14 @@ module.exports = ({ strapi }) => {
           Array.isArray(entries) &&
           typeof contentTypeConfig?.filterEntry === 'function'
         ) {
-          const filtered = entries.filter(entry =>
-            contentTypeConfig.filterEntry({
+          const filtered = await entries.reduce(async (memo, entry) => {
```

Comment:
I think we can return without having to declare the `const` right?

```javascript
    return await entries.reduce(async (memo, entry) => {
        const result = await contentTypeConfig.filterEntry({
          entry,
          contentType,
        })

        if (!result) return memo

        return [...(await memo), entry]
      }, [])
```

Reply:
I prefer leaving this temp variable so that I can add actions beneath it when needed or for easy debug. I also like the fact that the `return` keyword` is very easy to spot.